src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -65,7 +66,7 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
                 CultureUtility.DisableLocalization();
             }
 
-            log.LogLevel = LogLevel.Information;
+            log.VerbosityLevel = LogLevel.Information;
```

Comment:
Won't this change it for all commands? We don't (currently) want to break compatibility for existing commands.

Reply:
This would change other `dotnet` commands as well. I have provided some context about why I made this change in the [PR description](https://github.com/NuGet/NuGet.Client/pull/3662#issue-486225798). If this is considered as risk now, I will revert it. I should find another approach if we need to display `NuGet Log code` in the error message to have parity with `nuget verify` error messages.

Reply:
you are correct Nikolche. removed this line in the next commit.